docs/UserGuide.md

```diff
@@ -182,11 +195,11 @@ Format: `exit`
 
 ### Adding a volunteer: `/v new`
 
-Adds a volunteer to the database.
+Welcome a new volunteer to your team by adding their details to the database.
 
 Format: `/v new n/NAME p/PHONE_NUMBER em/EMAIL d/AVAILABLE_DATE`
 
-Input Requirements:
+What you'll need:
 - Name: Alphanumeric and spaces only, 1-100 characters.
 - Phone Number: Digits only, 3-15 characters.
 - Email: Must follow local-part@domain format.
```

Comment:
Make that a bit more readable? I think its not the common terms used. Smth like username@domain?

Reply:
resolved